spec/services/idv/profile_maker_spec.rb

```diff
@@ -172,8 +172,8 @@
         let(:selfie_check_performed) { true }
 
         before do
-          allow(FeatureManagement).to receive(:idv_block_biometrics_requests?).
-            and_return(false)
+          expect(FeatureManagement).to receive(:idv_allow_selfie_check_in_login_prod?).
```

Comment:
Out of curiosity, why did you change `allow` to `expect` here?  I'm asking because I'm curious, not because I'm against the change.

Reply:
I think the specs should all be 'expect', because it's important to us that the check is happening. But it can be finicky to change that in a before block because sometimes it gets called twice in some of the specs for whatever reason. So I only changed the ones that worked when I changed it to expect. Now that I'm typing this, I could change the rest to .at_least_once or something like that. I'll look into it.